axes/tests.py

```diff
@@ -111,27 +110,27 @@ def test_failure_limit_many(self):
         """Tests the login lock trying to login a lot of times more
         than failure limit
         """
-        for i in range(1, FAILURE_LIMIT):
+        for i in range(1, axes_settings.FAILURE_LIMIT):
             response = self._login()
             # Check if we are in the same login page
             self.assertContains(response, self.LOGIN_FORM_KEY)
 
         # So, we shouldn't have gotten a lock-out yet.
         # We should get a locked message each time we try again
-        for i in range(0, random.randrange(1, FAILURE_LIMIT)):
+        for i in range(0, random.randrange(1, axes_settings.FAILURE_LIMIT)):
             response = self._login()
             self.assertContains(response, self.LOCKED_MESSAGE, status_code=403)
 
-    @patch('axes.decorators.cache.set', return_value=None)
```

Comment:
Maybe the one who did it (I wasn't) wanted to test the method setting the cache.

Feel free to improve it.